src/components/Inputs/ZipInput/ZipInput.js

```diff
@@ -20,13 +21,7 @@ export const ZipInput = (props) => {
   const [getError, setError, validate] = useErrorMessage(validator)
   const [touched, setTouched] = useState(false)
 
-  const setErrorWrapper = (value, errorValue) => {
-    if (!!formChangeHandler) {
-      formChangeHandler(value, errorValue)
-    }
-    setError(errorValue)
-  }
-
+  // This has to come before useInputValidation setup below
```

Comment:
Only since it's an error function :) Another pattern would be to use a regular ole' `function` and then you wouldn't have to order these. But I really really prefer arrow functions when at all possible to use :)